python/letsql/backends/let/datafusion/compiler.py

```diff
@@ -576,6 +576,9 @@ def visit_GroupConcat(self, op, *, arg, sep, where, order_by):
     def visit_ArrayFlatten(self, op, *, arg):
         return self.if_(arg.is_(NULL), NULL, self.f.flatten(arg))
 
+    def visit_Hash(self, op, *, arg):
+        return self.f.hash_int(arg)
```

Comment:
I see the rust implementation of `hash_int` , how does `hash_int` end up becoming accessible by our data fusion compiler?  in `src/context.rs`

Reply:
@mesejo i think the expectation of `visit_Hash` is that it returns a string so we should rename this to something else

Reply:
@mesejo , nvm, per @ghoersti  all the other `visit_Hash`s ***do*** return int